ci/common/submit_coverage.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
```

Comment:
`upload_coverage.sh` would be more consistent with other script names:

```
  third-party/cmake/DownloadAndExtractFile.cmake
  scripts/download-unicode-files.sh
 cmake/Download.cmake
```

Reply:
I've used the previous function name.
I think it is better fitting, because it not only uploads, but also generates it.
Easy to change though, of course.